src/persistence/graph6.jl

```diff
@@ -0,0 +1,148 @@
+
+function _bv2int(x::BitVector)
+  assert(length(x) <= 8 * sizeof(Int))
+  acc = 0
+  for i = 1:length(x)
+    acc = acc << 1 + x[i]
+  end
+  return acc
+end
+
+function _int2bv(n::Int, k::Int)
+  bitstr = lstrip(bits(n), '0')
+  l = length(bitstr)
+  padding = k - l
+  bv = falses(k)
```

Comment:
I'd rather not encourage docstrings for private methods. :) I can add a comment.